test/rubygems/test_gem_commands_update_command.rb

```diff
@@ -373,7 +373,13 @@ def test_execute_user_install
     installer = @cmd.installer
     user_install = installer.instance_variable_get :@user_install
 
-    assert user_install, 'user_install must be set on the installer'
+    assert user_install, "user_install must be set on the installer"
```

Comment:
Usual pickiness of mine of not changing unnecessary code just for style.

Reply:
🤔 I usually don't, unless is related  code that I'm modifying any way 🙈

Reply:
Right. Although I prefer to never touch style unless explicitly normalizing it, I'm also aware of that approach of "fix style as you go".

I won't bug you with this kind of comments any more if the code you're changing contains minor style tweaks next to it :+1:.